tests/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.17
 require (
 	github.com/phoenixnap/go-sdk-bmc/auditapi v1.0.2
 	github.com/phoenixnap/go-sdk-bmc/ipapi v1.1.0
+	github.com/phoenixnap/go-sdk-bmc/ranchersolutionapi v1.1.1
```

Comment:
The version here is actually `v1.1.2` - could you fix the other versions since they don't seem to match as well?

Reply:
isnt this the last tag for rancher solution api? [ranchersolutionapi](https://github.com/phoenixnap/go-sdk-bmc/releases/tag/ranchersolutionapi%2Fv1.1.1)

Reply:
resolved offline